tests/test_solvers_fairflow.py

```diff
@@ -68,7 +156,7 @@ def test_solver_impossible_constraints():
     )
 
     with pytest.raises(SolverException):
-        solution = solver.solve()
+        _ = solver.solve()
```

Comment:
if `solution` is not being used then don't assign the result to anything.

Reply:
Agreed. Change has been made.